src/main/scala/chisel3/testers/TesterDriver.scala

```diff
@@ -62,8 +107,9 @@ object TesterDriver extends BackendCompilationUtilities {
         writeResourceToDirectory(name, path)
       })
 
-      (new FirrtlStage).execute(Array("--compiler", "verilog"), annotationsx)
```

Comment:
`annotations` may be dead code now.

Reply:
I'm not sure what this comment means.

Reply:
he's saying that because you no longer pass `annotationsx` to anything, why even compute it.